frontend/packages/console-shared/src/components/dashboard/activity-card/ActivityBody.tsx

```diff
@@ -112,13 +112,20 @@ export const RecentEventsBodyContent: React.FC<RecentEventsBodyContentProps> = (
         ))}
       </Accordion>
       {sortedEvents.length > 50 && !!moreLink && (
-        <Link
-          className="co-activity-card__recent-more-link"
-          to={moreLink}
-          data-test="events-view-all-link"
-        >
-          {t('console-shared~View all events')}
-        </Link>
+        <>
+          <Divider />
+          <Card isPlain>
```

Comment:
still not sure how I feel about nesting a plain Card in another Card -- can we try rearranging the elements in a way that would remove the need for this?